c-pallets/file-map/src/lib.rs

```diff
@@ -127,12 +138,65 @@ pub mod pallet {
 	#[pallet::storage]
 	#[pallet::getter(fn bond_acc)]
 	pub(super) type BondAcc<T: Config> =
+	// Same as for SchedulerMap. Wrong use of StringLimit. Needs new constant
```

Comment:
This storage should use the same upper limit as the `SchedulerMap`:
https://github.com/CESSProject/cess/pull/88/files#diff-fba8d9a7246fa1a7499263ade4e960d909c56ddf5a3a423d7c763c423ef987b1R119-R133